pkg/daemon/ceph/client/status.go

```diff
@@ -158,6 +158,23 @@ func Status(context *clusterd.Context, clusterName string) (CephStatus, error) {
 	return status, nil
 }
 
+func StatusExternal(context *clusterd.Context, clusterName, userName string) (CephStatus, error) {
```

Comment:
How about naming this `StatusWithUser()`? It's not really specific to external, that's just the first scenario where we're using it.